### DISCUSSION TITLE
Aide

### DISCUSSION THREAD

**Discussion by GTnkh** [community/community#47487] [New to GitHub] [answered]
Bonjour je suis nouveau sur github , j'aimerais en savoir plus svp merci 😊

**Comment by queenofcorgis** [comment, marked as answer]
Welcome to GitHub and the GitHub Community @GTnkh  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
